indico/modules/events/papers/lists.py

```diff
@@ -97,6 +101,19 @@ def _filter_list_entries(self, query, filters):
             if state_criteria:
                 criteria.append(db.or_(*state_criteria))
 
+        if 'unassigned' in filters['items']:
+            role_map = {
+                PaperReviewingRole.judge.value: Contribution.paper_judges,
+                PaperReviewingRole.content_reviewer.value: Contribution.paper_content_reviewers,
+                PaperReviewingRole.layout_reviewer.value: Contribution.paper_layout_reviewers,
+            }
+            filtered_roles = filters['items']['unassigned']
+            unassigned_criteria = []
+            for filter_role in filtered_roles:
+                unassigned_criteria.append(role_map[int(filter_role)] == None)
```

Comment:
I wanted to do it this way at the beginning but I was forgetting the `.any()`, thanks.